src/main/java/bowling/view/ResultView.java

```diff
@@ -0,0 +1,135 @@
+package bowling.view;
+
+import bowling.domain.Player;
+import bowling.domain.Score;
+import bowling.domain.frame.Frame;
+import bowling.domain.frame.FrameRound;
+import bowling.domain.frame.Pin;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class ResultView {
+    private static final String HEADER = "| NAME |  01  |  02  |  03  |  04  |  05  |  06  |  07  |  08  |  09  |  10  |";
+    private static final int EACH_STRING_MAX_SIZE = 6;
+    private static final String SEPARATOR = "|";
+    private static final String BLANK = " ";
+
+    public static void printScore(String player, List<Frame> frames) {
+        List<String> result = new ArrayList<>();
+        result.add(player);
+
+        frames.stream()
+                .filter(frame -> !frame.isFinal())
+                .filter(frame -> frame.getPinsSize() > 0)
+                .map(ResultView::getNormalFrame)
+                .forEach(result::add);
+
+        Frame finalFrame = frames.get(FrameRound.MAX_ROUND - 1);
+        result.add(getFinalFrame(finalFrame));
+        printResult(result);
+    }
+
+    public static void printResult(List<String> results) {
+        System.out.println(HEADER);
+        printGameResult(results);
+    }
+
+    private static void printGameResult(List<String> result) {
+        StringBuilder sb = new StringBuilder();
+        IntStream.range(0, 11)
+                .forEach(i -> sb.append(SEPARATOR).append(getResult(result, i)));
+        System.out.println(sb + SEPARATOR) ;
+    }
+
+    private static String getResult(List<String> results, int index) {
+        if (results.size() <= index) {
+            return getCenterFormat("", EACH_STRING_MAX_SIZE);
+        }
+        return getCenterFormat(results.get(index), EACH_STRING_MAX_SIZE);
+    }
+
+    public static String getCenterFormat(String str, int size) {
+        if (str == null) {
+            str = "";
+        }
+        return getCenterAligned(str, size);
+    }
+
+    private static String getCenterAligned(String str, int size) {
+        StringBuilder sb = new StringBuilder(size);
+        appendPad(sb, (size - str.length()) / 2);
+        sb.append(str);
+        appendPad(sb, (size - sb.length()));
+        return sb.toString();
+    }
+
+    private static void appendPad(StringBuilder sb, int range) {
+        IntStream.range(0, range)
+                .forEach(index -> sb.append(BLANK));
+    }
+
+    private static String getFinalFrame(Frame frame) {
+        if (!(frame.getPinsSize() > 0)) {
```

Comment:
```suggestion
        if (frame.getPinsSize() <= 0) {
```
이렇게 표현하면 not 연산이 없으니 좀더 읽기 좋은것 같은데 어떤가요? 🙂